### DISCUSSION TITLE
Question regarding Zowe Installation 2.8 and ports

### DISCUSSION THREAD

**Discussion by TitanNitWeb** [zowe/docs-site#2889] [Q&A] [answered]
Hi,

I have recently installed Zowe 2.8.0 and now have a problem where I find nothing suitable in the docs.
Zowe is up and running and the API-Gateway and apis are working fine - but the JES/IP/MVS/USS Explorer are trying to access it on the port of the APP-Server.

So calling https://myserver:8544/ZLUX/plugins/org.zowe.zlux.bootstrap/web/ works fine and using the apis too: https://myserver:7554/gateway/api/v1/version - but once I try to open any Explorer it calls in the background: https://myserver:8544/gateway/api/v1/auth/query or https://myserver:8544/gateway/api/v1/auth/login - So is this a bug or a miss-configuration?

Here the configuration.yaml file (skipped some parts):

` externalPort: 7554` 
  ` components:
  gateway:
  enabled: true  
  port: 7554  
  debug: false`
   `apiml:
 security:
   auth:  
 provider: zosmf 
 zosmf:   
   jwtAutoconfiguration: auto  
   serviceId: zosmf   
   authorization: endpoint:
   enabled: false
 provider: ""  
   x509:  
 enabled: false
    server:    
 internal:  
   enabled: true   
   port: 7550 
   ssl:   
 enabled: false`
`metrics-service:
   enabled: true 
   port: 7551    
   debug: false`
 `cloud-gateway:  
   enabled: true 
   port: 7563    
   debug: false`
 `api-catalog:    
   enabled: true 
   port: 7552    
   debug: false`
`discovery: 
   enabled: true 
   port: 7553    
   debug: false  
   enabled: true   
   port: 7555 
   debug: false`      
 `app-server:  
   enabled: true   
   port: 8544`
 `zss:    
   enabled: true
  port: 7557   
  crossMemoryServerName: ZWESIS_STD
  tls: true    
  agent:
    jwt:
 fallback: true`    
   
`jobs-api: 
  enabled: true
  debug: false 
  port: 7558`  
   
`files-api:
  enabled: true
  debug: false 
  port: 7559` 
   
`explorer-jes:  
  enabled: true`
   
`explorer-mvs:  
  enabled: true`
   
`explorer-uss:  
  enabled: true`

Thanks and best regards,
Daniel

**Comment by boris-bc** [comment, marked as answer]
@TitanNitWeb , you should be accessing the app framework via the gateway port only, not directly. In your case, the endpoint would look like: `https://myserver:7554/zlux/ui/v1/ZLUX/plugins/org.zowe.zlux.bootstrap/web/`. Calling it directly on port 8544 is not supported in V2. Please let me know if that would solve the problem.

**Comment by DivergentEuropeans** [comment]
👍 to @boris-bc 's point. For more info, check out: https://docs.zowe.org/stable/user-guide/mvd-configuration#accessing-the-desktop

**Comment by DivergentEuropeans** [comment]
In the future, there will be an automatic re-routing to the correct URL but not implemented yet
